tests/test_merge.py

```diff
@@ -1,8 +1,75 @@
 import unittest
+from click.testing import CliRunner
+from os import path
+from pathlib import Path
+import os
+import shutil, tempfile
+import pandas as pd
+
+from quantaq_cli.console import merge
+
 
 class SetupTestCase(unittest.TestCase):
     def setUp(self):
```

Comment:
Just a question on what this does? just the "setUp(self)," is it the same as "__init__"

Reply:
The python unittest library, which is used here, requires two methods: `setUp` and `tearDown` which allow you to run custom code that is evaluated before and after every test.